api/queries_sponsors.go

```diff
@@ -0,0 +1,90 @@
+package api
+
+import "net/http"
+
+type SponsorQuery struct {
+	User User `json:"user" graphql:"user(login: $username)"`
+}
+
+type User struct {
+	Sponsors Sponsors `json:"sponsors" graphql:"sponsors(first: 30)"`
+}
+
+type Sponsors struct {
+	TotalCount int    `json:"totalCount"`
+	Nodes      []Node `json:"nodes"`
+}
+
+type Node struct {
+	Login string `json:"login"`
+}
```

Comment:
In your comment below about query via reflection / field tags, you could look at the organization code for an example:

https://github.com/cli/cli/blob/8da9ee2437ee95a39085ede1006f22283526d7fc/api/queries_org.go#L11-L21

Reply:
AMAZING! I was having issues getting the graphql strings correct and had to resort to this implementation. I'll see if I can piece it together from here 🙏